gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,3 +1,10 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
```

Comment:
Anything in `gradle/wrapper` folder can be ignored.

Reply:
Do you want me to ignore the folder or make the comments in gradle/wrapper uniform?

Reply:
Ignore `gradle/wrapper` and make other license comments uniform by changing `\\` to `#`

Reply:
Added license comments as discussed over Slack.